docs/docs/en/guide/metrics/metrics.md

```diff
@@ -0,0 +1,72 @@
+# Introduction
+
+Apache DolphinScheduler has export some metrics to monitor the system. We use micrometer for the exporter facade, and
+the default exporter is prometheus, more exporter is coming soon.
+
+## Quick Start
+
+You can add the following config in master/worker's yaml file to open the metrics exporter.
+
+```yaml
+metrics:
+  enabled: true
+```
+
+Once you open the metrics exporter, you can access the metrics by the url: `http://ip:port/actuator/prometheus`
```

Comment:
Can you please add some additional information about the ports of each service, 5679 for master instead of 5678, 1235 for worker instead of 1234? Users might think the port of metrics endpoint is the same as the service, which is wrong

Reply:
Done